src/components/Reports/AdditionalSalaryRequest/SharedComponents/NavButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Button, ButtonProps, Tooltip } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+
+type NavButtonType = 'continue' | 'cancel';
+
+interface NavButtonProps {
+  type: NavButtonType;
+  onClick: () => void;
+}
+
+const StyledButton = styled(Button)(({ theme }) => ({
+  padding: theme.spacing(1, 4),
+}));
+
+export const NavButton: React.FC<NavButtonProps> = ({ type, onClick }) => {
+  const { t } = useTranslation();
+
+  const config: Record<
```

Comment:
This lookup feels complex for just having two button types. I wonder if we should just do:

```tsx
if (type === 'continue') {
  return <Tooltip title={t('Proceed to the next section. Your progress is automatically saved as you go.')}>
    <StyledButton
      variant="contained"
      color={color}
      size="large"
      onClick={onClick}
    >
      {t('Continue')}
    </StyledButton>
  </Tooltip>;
} else if (type === 'cancel') {
  return ...
}
```